### PR TITLE
Remove latest-version script

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
This is the forward-port of the changes we did for the release-1.18
branch as well.
#### Which issue(s) this PR fixes:

Refers to https://github.com/cri-o/cri-o/pull/3613 and https://github.com/cri-o/cri-o/pull/3621
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: internal/version/version.go [hunk 1]
```diff
@@ -19,9 +19,11 @@ import (
 	"github.com/cri-o/cri-o/utils"
 )
 
+// Version is the version of the build.
+const Version = "1.19.0"
```
Comment (haircommander) [inline internal/version/version.go:23]: I think for now this should be `1.19.0-dev` to make clear we haven't released any 1.19  yet

**Comment by TomSweeneyRedHat** [comment]
Other than @haircommander 's comment, LGTM

File: internal/version/version.go (hunk 1 above)
Comment (saschagrunert) [reply internal/version/version.go]: Fixed :)

**Pull request merged**